cli/src/cmd_auth.rs

```diff
@@ -535,6 +519,78 @@ impl CmdAuthStatus {
     }
 }
 
+/// Rename a profile.
+///
+/// This command does not modify the profile credentials. If
```

Comment:
why start with this? Might people think that it would?

```suggestion
/// If
```